assignments/function-conversion.js

```diff
@@ -1,30 +1,27 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction = () => console.log("Function was invoked!");
+ 
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = (param) => {return param;
+};
```

Comment:
If we only pass one parameter we can omit the parenthesis and if we only have one line returning we can also omit the curly brackets and the return keyword add the arrow function will implicitly add these for us, that could look like this. let anotherFunction = params => params;